setup.cfg

```diff
@@ -34,6 +34,7 @@ pycuda =
 
 test =
     pytest
+    pytest-custom_exit_code
```

Comment:
A couple of points.
1. How did you deduce you needed this?
2. The project does not seem to be too... mature (or even maintained?). I ask this genuinely: Is it wise to add this dependency?
3. Speaking of active development, there is nothing explicit that says this package should/will work with `ska-sa/vkgdr`
  *  The package listing itself indicates support for Python 3.7, while
  * `ska-sa/vkgdr` requires >= 3.9
  * Did you need to do any tweaking to get it to work?